bundles/org.openhab.binding.govee/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Why is this file located in /update and called instructions.xml? 
Answer to myself (and any reader): I found when you were asking for that 😅
https://community.openhab.org/t/documentation-of-upgrade-instructions-xml/160338

Reply:
As you discovered the update instructions xml file is a mechanism that tells OH core how to update things and respective items that had been created on a previous thing xml description to a new xml description. In this case we tell it that the color temperature absolute channel is now based on the system:color-temperature-abs channel type